<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestSettings</class>
 <widget class="QWidget" name="TestSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Test settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="9">
      <widget class="QPushButton" name="addValue1">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="2" column="9" colspan="2">
      <widget class="QLabel" name="label_value1">
       <property name="text">
        <string>Value 1</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="11">
      <widget class="QLabel" name="labelTestSettingsDescription">
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
     </item>
     <item row="2" column="8" rowspan="3">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="addMp1SmuAddress">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="3" column="9" colspan="2">
      <widget class="QListWidget" name="value1_list">
       <item>
        <property name="text">
         <string>0</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QPushButton" name="removePsmuAddress">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" rowspan="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="6">
      <widget class="QPushButton" name="addValue0">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="3" column="6" colspan="2">
      <widget class="QListWidget" name="value0_list">
       <item>
        <property name="text">
         <string>-1000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>100</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1000</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="addPsmuAddress">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="4" column="7">
      <widget class="QPushButton" name="removeValue0">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="11">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelRange1">
         <property name="text">
          <string>Address or value range: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="minRangeValue">
         <property name="minimum">
          <number>-214748368</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelRange2">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="maxRangeValue">
         <property name="maximum">
          <number>2147483647</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="labelMp1SmuAddresses">
       <property name="text">
        <string>Mp1 SMU addresses</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3" colspan="2">
      <widget class="QLabel" name="labelPsmuAddresses">
       <property name="text">
        <string>PSMU addresses</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="removeMp1SmuAddress">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="4" column="10">
      <widget class="QPushButton" name="removeValue1">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="11">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="loadTestSettings">
         <property name="text">
          <string>Load</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="saveTestSettings">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="5" rowspan="3">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QListWidget" name="mp1smu_list">
       <item>
        <property name="text">
         <string>0</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="6" colspan="2">
      <widget class="QLabel" name="label_value0">
       <property name="text">
        <string>Value 0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3" colspan="2">
      <widget class="QListWidget" name="psmu_list"/>
     </item>
     <item row="1" column="0" colspan="11">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="labelStatisticPeriod">
         <property name="text">
          <string>One value statistic period</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="statisticPeriod">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
